README.md

```diff
@@ -1,4 +1,92 @@
 # CDM Jupyterhub dockerfiles (Prototype)
 
 This prototype establishes a Docker container configuration for JupyterHub, designed to furnish a multi-user 
-environment tailored for executing Spark jobs via Jupyter notebooks.
\ No newline at end of file
+environment tailored for executing Spark jobs via Jupyter notebooks.
+
+## Using `docker-compose.yaml`
+
+To deploy the JupyterHub container and Spark nodes locally, execute the following command:
+
+```bash
+docker-compose up --build
```

Comment:
You can daemonize with `-d`

Reply:
I know. But I usually want to see the logs.

Reply:
`docker compose logs`

but if you want to leave it as is that's fine

Reply:
I know you can view logs that way. I think it's convenient to view it as it's happening. I will just leave it as is for now. It's individual dev's choice I guess.